### PR TITLE
docs for stress perf

### PR DESCRIPTION
**Pull request by gearama**
# Pull Request Checklist

Please leverage this checklist as a reminder to address commonly occurring feedback when submitting a pull request to make sure your PR can be reviewed quickly:

See the detailed list in the [contributing guide](https://github.com/Azure/azure-sdk-for-cpp/blob/main/CONTRIBUTING.md#pull-requests).

- [ ] [C++ Guidelines](https://azure.github.io/azure-sdk/cpp_introduction.html)
- [ ] Doxygen docs
- [ ] Unit tests
- [ ] No unwanted commits/changes
- [ ] Descriptive title/description
  - [ ] PR is single purpose
  - [ ] Related issue listed
- [ ] Comments in source
- [ ] No typos
- [ ] Update changelog
- [ ] Not work-in-progress
- [ ] External references or docs updated
- [ ] Self review of PR done
- [ ] Any breaking changes?

### CODE REVIEW

**Review by LarryOsterman — APPROVED** [review]

**Comment by ahsonkhan** [comment]
@gearama **Please don't merge PRs without having CI runs complete (at least once).**

The link verification step from this PR is failing, which caused main to be broken and other PRs are impacted.

In the future contact EngSys if check-enforcer or license/cla is stuck. I added you to a conversation about that, it is impacting other repos too.

**Comment by ahsonkhan** [comment]
Why do we have an empty `/doc/TestProxy.md` file checked-in?

File: CONTRIBUTING.md
Comment (ahsonkhan) [inline CONTRIBUTING.md:70]: This link shouldn't have `en-us`. The link verification fails because of it.
```suggestion
Required to get and execute test proxy(see /doc/TestProxy.md). [website](https://learn.microsoft.com/dotnet/core/tools/dotnet)
```

Also, the link isn't the most appropriate for finding where to get `dotnet` from.

Resolved in https://github.com/Azure/azure-sdk-for-cpp/pull/4273

**Review by ahsonkhan — COMMENTED** [review]
@gearama Please fix some of the typos or spelling mistakes in the doc. I mentioned a few of them, but it would be good to run it through a spell-check/another read through to fix the remaining ones.

Since you own this, rather than fixing myself, I'll leave it to you for now to address as you see fit :)

**Pull request merged**
